docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -226,8 +226,45 @@ Note also that the main navigation logo uploaded via the admin panel supersedes
 :::
 
 #### Favicon
+Replacing the Strapi favicon with a custom favicon. 
+:::tip
+This same process can be used to replace the login logo `AuthLogo` and menu logo with `MenuLogo`. For more details, please visit Strapi [#logos](#logos)
+:::
+
+1. Create an extensions folder in: 
```

Comment:
Not sure if this step is required, same with step 2 :thinking: you could instead import the favicon from the root and not need to use extensions at all.

Reply:
the CLA says it is signed for me. 

I tested without the extensions folder and default Strapi app populated; like the fig I shared initially.

Reply:
oh, i see what you mean. Good idear

Reply:
> Not sure if this step is required, same with step 2 thinking you could instead import the favicon from the root and not need to use extensions at all.

Does Strapi users have to use favicon.ico format? I prefer to use .svg or .png.